elastic/kibana
# Fixing DarkMode issue on Security Plugin CodeEditorFields

## Summary

Closes https://github.com/elastic/kibana/issues/175078

## Testing

Instructions:

1. Login as `elastic`
2. Navigate to User Profile, enable DarkMode
3. Navigate to the following screens and check CodeEditorFields for appropriate themeing:
  - API Key Flyout
  - Edit Roles page
  - Role Mappings

## Screenshots
<img width="1294" alt="Screenshot 2024-01-23 at 9 52 02 AM" src="https://github.com/elastic/kibana/assets/21210601/96d9e58f-647d-4efc-87f3-b7cf03cf25b3">
<img width="704" alt="Screenshot 2024-01-23 at 9 52 15 AM" src="https://github.com/elastic/kibana/assets/21210601/7776a7e3-6c5a-40d1-9cd0-ff702998977d">
<img width="1100" alt="Screenshot 2024-01-23 at 10 47 00 PM" src="https://github.com/elastic/kibana/assets/21210601/fe0c8fef-0959-41bf-8351-eb1f3350a93b">
